### PR TITLE
move hardcoded route to default host/port

### PR DESCRIPTION
**Pull request by supermarin**
This should allow us to specify a custom API route without getting `/api/v0/reports` appended.

### CODE REVIEW

File: Pod/Classes/LSTracer.m [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #import "LSUtil.h"
 #import "LSVersion.h"
 
-NSString* const LSDefaultHostport = @"collector.lightstep.com:443";
+NSString* const LSDefaultHostport = @"collector.lightstep.com:443/api/v0/reports";
```
Comment (bhs) [inline Pod/Classes/LSTracer.m:11]: `LSDefaultHostportPath`

Comment (bhs) [reply Pod/Classes/LSTracer.m:11]: ... and we should change the comments and internal variable name accordingly, too.

**Review by bhs — COMMENTED** [review]
@supermarin I am happy to make these tiny nits in a follow-on change as you wish. Let me know what you prefer.

**Comment by supermarin** [comment]
cloisng in favor of discussed NSURL refactor

**Pull request closed without merging**